Config/macOS.xcconfig

```diff
@@ -1,6 +1,66 @@
 SDKROOT = macosx
+
+MACOSX_DEPLOYMENT_TARGET = 10.15
```

Comment:
Let's only change this for the app itself, and move all these additional flags into the app specific xcconfig